manifests/overlays/test/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+patches:
+  - patch: |-
+      - op: replace
+        path: /spec/template/spec/containers/0/env/3/value
+        value: development
```

Comment:
In test we probably want to run the production build/runtime. If the code differs for dev/prod node environment, we want rather see how the prod behaves in test.